recipes/reprozip/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "reprozip" %}
+{% set version = "1.0.9" %}
+{% set sha256 = "20412ac156fb8cdbc62a26890f5dd92d403c923755d84d739e1316cac252dd2b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [not linux32 and not linux64]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - sqlite
```

Comment:
Setting those seems to help! Looks like a bug in your setup? (conda-build or conda-forge I can't say)